lib/project_types/script/graphql/app_script_set.graphql

```diff
@@ -10,6 +10,8 @@ mutation AppScriptSet(
   $configurationUi: Boolean!,
   $configurationDefinition: String!,
   $moduleUploadUrl: String!,
+  $libraryLanguage: String!,
+  $libraryVersion: String!,
```

Comment:
So I've made these fields optional on the Script Service side, they just default to nil if nothing is passed.  

For this PR tho, I feel like it's still correct to leave these here as required? Old CLIs will just not pass these variables but we want the new CLIs to include it. Does that make sense?

Reply:
>For this PR tho, I feel like it's still correct to leave these here as required?

In my comment, I was thinking we shouldn't make these required so it more accurately reflects what Script Service has. But if we're going to change it to required relatively soon, then I don't think there's any harm in leaving it required here for now (assuming it can successfully make requests).

Reply:
I reworked this mutation definition since I changed some things on the script-service side.
I've made the new parameter optional, since that matches script service's current behaviour.